app/assets/javascripts/locationForm.js

```diff
@@ -1,129 +1,134 @@
 function addGrid(map)
- {
-    L.VirtualGrid = L.FeatureGroup.extend({
-      include: L.Mixin.Events,
-      options: {
-        cellSize: 64,
-        delayFactor: 2.5,
-        style: {
-          stroke: true,
-          color: '#3ac1f0',
-          dashArray: null,
-          lineCap: null,
-          lineJoin: null,
-          weight: 2,
-          opacity: 1,
+ {function create_cell_style(fill) {
```

Comment:
This looks good but could you reformat a bit using the following style:

```js
function addGrid(map) {
  // code here
}
```

That way things stay a bit more readable and consistent. Thank you!